api/v1alpha1/imagespecjob_types.go

```diff
@@ -33,6 +33,8 @@ type ImageSpecJobSpec struct {
 	TargetImage string                `json:"targetImage"`
 	PushSecret  string                `json:"pushSecret"`
 	RepoPrefix  string                `json:"repoPrefix"`
+
+	UpdateTime string `json:"updateTime"`
```

Comment:
Just would like to make sure the updateTime logic

Currently, the implementation of the MR is, if `imagespec.updateTime !=  imagespecjob.upodateTime`, delete and recreate the job, that is rerun the job against the hash.  But i am not sure if it is what we need.  Because it will always trigger the image building process.

Pro
1. We provide a way to rebuild the image. For admin, just save the imagespec

Cons
1. It will always rebuild the image even though the spec is not change.

I think the behavior acceptable for me, but would like to make sure if it is what we want.

Reply:
Type is changed to `metav1.Time`

It seems to me that the logic of `imagespec.updateTime` is fine. `imagespec.updateTime` is designed for rebuilding and it's not about UI, so the UI shouldn't update `updateTime` each time. For now, always rebuilding is not a problem.